scripts/jenkins/qa/cloud-mkphyscloud-qa-scenario-1a.yaml

```diff
@@ -39,6 +39,11 @@
           description: Optional, specify cct tests to run
 
       - string:
+          name: integration
```

Comment:
IMO bad choice of a parameter name, `integration_tests` would be better. 
As the PR description is missing, not sure how this change is meant; should it cover also other integration tests for openstack or it's horizon specific? If it's horizon specific, then `horizon_tests` would be sufficient.